shell/models/__tests__/management.cattle.io.node.ts

```diff
@@ -0,0 +1,82 @@
+import MgmtNode from '@shell/models/management.cattle.io.node';
+
+describe('class MgmtNode', () => {
+  const t = jest.fn(() => 'baz');
+  const ctx = { rootGetters: { 'i18n/t': t } };
+
+  const resetMocks = () => {
+    // Clear all mock function calls:
+    jest.clearAllMocks();
+  };
+
+  it('should not return addresses if they are not present in the resource status, the internalNodeStatus, or the rkeNode key in status', () => {
+    const mgmtNode = new MgmtNode({ status: {} }, ctx);
+
+    expect(mgmtNode.addresses).toStrictEqual([]);
+    resetMocks();
+  });
+  it('should return addresses if they are present directly on the resource status', () => {
+    const mgmtNode = new MgmtNode({ status: { addresses: ['foo'] } }, ctx);
```

Comment:
Repeated value `['foo']` should be a constant to increase readability. Please consider all the other cases too, so I don't have to go one by one.

Reply:
I feel like this is perhaps gilding the lily but it's a simple change so your wish is my command.

For reference, you never have to go one by one on these, if you mention that something is a problem and I go about fixing it, I'll change it everywhere I think it's relevant.